helpers/config.py

```diff
@@ -293,17 +316,32 @@ def write_config(self):
         self.__config["date_modified"] = int(time.time())
 
         try:
-            with open(Config.CONFIG_FILE, "w") as f:
+            base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+            config_file = "{}/{}".format(base_dir, Config.CONFIG_FILE)
+            with open(config_file, "w") as f:
                 f.write(json.dumps(self.__config))
 
-            os.chmod(Config.CONFIG_FILE, stat.S_IWRITE | stat.S_IREAD)
+            os.chmod(config_file, stat.S_IWRITE | stat.S_IREAD)
 
         except Exception as e:
             CLI.colored_print("Could not write configuration file", CLI.COLOR_ERROR)
             return False
 
         return True
 
+    def write_unique_id(self):
+        try:
+            unique_id_file = "{}/{}".format(self.__config.get("kobodocker_path"), Config.UNIQUE_ID_FILE)
+            with open(unique_id_file, "w") as f:
+                f.write(str(self.__config.get("unique_id")))
+
+            os.chmod(unique_id_file, stat.S_IWRITE | stat.S_IREAD)
+        except Exception as e:
```

Comment:
Which exceptions? `IOError` and `OSError`?

Reply:
Right!